app/views/posts/_form_new_post.html.erb

```diff
@@ -0,0 +1,11 @@
+<%= form_with model: @post, local: true, url: user_posts_path(@users),
+html: {class: "column form"} do |f| %>
+  <%= f.text_field :title, 
+    placeholder: "Title",
+    maxlength: 250,
+    required: true
+  %>
+  <%= f.label :title, "maximum 250 characters" %>
+  <%= f.text_area :text, placeholder: "text ...", required: true %>
+  <%= f.submit "Create Post" %>
+<% end %>
```

Comment:
https://user-images.githubusercontent.com/65068771/231975500-582d7140-eadc-4702-a5d9-9bad8a936170.mov

- [ ] Kindly make sure the comment is created as well as liking a post, also clicking on a post or a user should bring you to the corresponding details. refer your self in the screencast above 👆

Reply:
Fixed all the functionality stated in the video above